cosmoz-moment.html

```diff
@@ -12,10 +31,10 @@
 		window.Cosmoz = window.Cosmoz || {};
 
 		var localSharedMoment = moment(),
-			locale = 'en',
```

Comment:
The last one. It was correct before the PR, see https://github.com/Neovici/cosmoz-moment/pull/2/files